scripts/test.sh

```diff
@@ -97,9 +98,16 @@ fi
 
 echo ">> Running CMake"
 if [[ $TEST_DRYRUN != 1 ]]; then
+    (
+    if [[ $TEST_CC = gcc ]]; then
+        export CC=gcc CXX=g++
+    elif [[ $TEST_CC = clang ]]; then
+        export CC=clang CXX=clang++
+    fi
+
     set -x
     cmake "${CMAKE_ARGS[@]}" "${GITREPO_ROOT}" || add_fatal_failure "cmake configure"
-    set +x
+    )
```

Comment:
Intentional removal of `set +x`?

Reply:
I set it to spawn a sub-shell (with parentheses), so that both variables are only exported inside it. It also applies to shell options, so the effect of `set -x` is contained as well.

Reply:
Ah, I see now. That's fine, then.